presets/consumer/avformat/Screencast-H264

```diff
@@ -0,0 +1,22 @@
+pix_fmt=yuv420p
+vcodec=libx264
+vprofile=high
+crf=27
+g=249
+bf=8
+preset=veryfast
+b-pyramid=0
+refs=8
+weightb=1
+b_strategy=2
+rc-lookahead=25
+channels=1
+acodec=aac
+ar=32000
+ab=48k
+f=mp4
+movflags=+faststart
+
+meta.preset.extension=mp4
+meta.preset.note=Small file size for screencasts, video conferences, and slide decks. Hardware encoding must be OFF.
```

Comment:
I am not keen on the "Hardware encoding must be OFF." It is no longer true that all hardware encoding is inferior and produces large sizes. Sure, you can make that claim in response to a forum question about a particular encoder's large output but not sure I like it here.

Reply:
Here's my dilemma... The H.264 version of the preset has customization that not even RTX can support.  The preset is heavily optimized for still images, giving a software encoder a clear advantage.  Non-RTX makes the preset almost pointless, as Quick Sync for example creates a file that is 5x as large with blurrier output at the same settings.  If somebody had hardware encoding on by default and used the preset with non-RTX, they would not be impressed with the results and would think the preset was ineffective.  Non-RTX still accounts for a lot of users today, especially those with laptop computers.

What about the phrase "Compare hardware encoding on and off to find the smallest file size."?

Reply:
How about "Hardware encoding not recommended."?

Reply:
I dig it.  Changes committed.